applications/jupyter-lab-hpc-native/tapisjob_app.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+echo "TACC: job ${SLURM_JOB_ID} execution at: $(date)"
+
+# TAP Port
+LOCAL_PORT=5902
+
+mkdir -p $PWD/Jupyter
+cd Jupyter
+
+# This file will be located in the directory mounted by the job.
+SESSION_FILE="delete_me_to_end_session"
+touch $SESSION_FILE
+
+# Make symlinks for work, home and scratch
+mkdir -p $PWD/Work $PWD/Home $PWD/Scratch
+if [ ! -L $PWD/Work ]; then
+    ln -s $STOCKYARD $PWD/Work
+fi
+if [ ! -L $PWD/Home ]; then
+    ln -s $HOME $PWD/Home
+fi
+if [ ! -L $PWD/Scratch ]; then
+    ln -s $SCRATCH $PWD/Scratch
+fi
```

Comment:
Let's include the work mount
```suggestion
if [ ! -L $PWD/Scratch ]; then
    ln -s $SCRATCH $PWD/Scratch
fi
if [ ! -L $PWD/MyData ]; then
    ln -s /data/designsafe/mydata/${_tapisJobOwner} $PWD/Scratch
fi
```

Reply:
a typo
`
ln -s /data/designsafe/mydata/${_tapisJobOwner} $PWD/Scratch
`
should be 
`
ln -s /data/designsafe/mydata/${_tapisJobOwner} $PWD/MyData
`

Reply:
I updated it to this: 

# Make symlinks for work, home, scratch, and mydata
mkdir -p $PWD/Work $PWD/Home $PWD/Scratch $PWD/MyData

if [ ! -L $PWD/Work ]; then
    ln -s $STOCKYARD $PWD/Work
fi

if [ ! -L $PWD/Home ]; then
    ln -s $HOME $PWD/Home
fi

if [ ! -L $PWD/Scratch ]; then
    ln -s $SCRATCH $PWD/Scratch
fi

if [ ! -L $PWD/MyData ]; then
    ln -s /data/designsafe/mydata/${_tapisJobOwner} $PWD/MyData
fi